R/exclusions.R

```diff
@@ -0,0 +1,99 @@
+#' Convert case counts in matching rows to NA
+#'
+#' NAs are skipped in model fitting by EpiNow2, so matched rows are excluded
+#' from the model likelihood.
+#'
+#' @param cases A dataframe returned by [read_data()]
+#' @param exclusions A dataframe returned by [read_exclusions()]
+#'
+#' @return A dataframe with the same rows and schema as `cases` and the value in
+#'   the column `confirm` converted to NA in any rows that match a row in
+#'   `exclusions`
```

Comment:
```suggestion
#' @return A dataframe with the same rows and schema as `cases` where the value in
#'   the column `confirm` is converted to NA in any rows which match a row in
#'   `exclusions`
```

Reply:
(Don't know how to do multi-line suggestions?)

Reply:
For the future, you click on one line and then do a drag and drop down the number of lines you want, like for a standard multiline comment. Then the suggestion box will auto-fill the multiple lines.

Reply:
But what is the actual suggestion here? I'm confused by the diff

Reply:
It's just swapping "and" to "where" basically -- add or ignore if you like it's tiny

Reply:
Oh! I see.